zaproxy/community-scripts
# Update Gradle to 4.10.2

Update Gradle and Spotless to 3.15.0.
Update to latest Kotlin DSL:
 - Use `getByName` to configure the `wrapper` task.
 - Use `sourceSets` directly.

Extract variable for JUnit version.
Add a comment to clarify the expected versions of the scripting
libraries.
